test/Debugger/MultipleContextStack.js

```diff
@@ -21,8 +21,8 @@ var script1 = WScript.LoadScript("\
   function Script1Func3() { scriptFunc3(); } \
   function setFunc2(func) { scriptFunc2 = func; } \
   function setFunc3(func) { scriptFunc3 = func; }",
-  "samethread", "Script1.js");

```

Comment:
Not necessarily. The comment there says; `// No fileName provided (WScript.LoadScript()), use dummy "script.js"` IMHO the intention there is clean. This particular script file doesn't do that